requirements.txt

```diff
@@ -29,20 +29,39 @@ future==0.15.2
 pyjwkest==1.0.1
 
 ### This section is for the actual libraries ###
-# these libraries are imported in code that can be reached via 
+# these libraries are imported in code that can be reached via
 # an executable that we ship
 
 # Flask must come after all other requirements that have "flask" as a
 # prefix due to a setuptools bug.
 Flask-Cors==2.0.1
 Flask==0.10.1
-avro==1.7.7
+
+coverage==3.7.1
+flake8==2.3.0  # Set due to conflict with pep8 in version 2.4.0
+freezegun==0.3.6
+guppy==0.1.10
 humanize==0.5.1
+mock==1.0.0
+nose==1.3.7
+pep8==1.6.2
 pysam==0.9.0
+PyVCF==0.6.7
+PyYAML==3.11
 requests==2.7.0
+snakefood==1.4
+Sphinx==1.3.1
 oic==0.7.6
 pyOpenSSL==0.15.1
 lxml==3.4.4
+protobuf==3.0.0b1.post1
+# Requirements for test OIDC provider.
+cherrypy==3.2.4
+pyaml==15.03.1
+Jinja2==2.7.3
+yubico-client==1.9.1
+pyjwkest==1.0.1
```

Comment:
Development dependencies and installation dependencies have been split into two files. This file currently contains duplicate entries causing the `pip install -r requirements.txt` to fail with double requirement errors. The entries only needed for development should be moved to the other file: `dev-requirements.txt`.